easybuild/easyconfigs/n/numactl/numactl-2.0.10-GCC-4.9.2.eb

```diff
@@ -11,6 +13,10 @@ The libnuma library provides convenient ways for you to add NUMA memory policies
 sources = [SOURCE_TAR_GZ]
 source_urls = ['ftp://oss.sgi.com/www/projects/libnuma/download/']
 
+dependencies = [('Automake', '1.15'),('libtool', '2.4.2')]
```

Comment:
put these on separate lines please? why is the libtool required exactly, is it used by `autogen`?
also, these are only build dependencies?

```python
builddependencies = [
    ('Automake', '1.15'),
    ('libtool', '2.4.2'),
]
```